src/main/java/us/kilroyrobotics/Constants.java

```diff
@@ -0,0 +1,321 @@
+
+        /** The radius of the elevator rope drum in Meters */
+        public static final Distance kElevatorDrumRadius = Meters.of(Inches.of(2.0).in(Meters));
+
+        /** The minimum height of the elevator in Meters */
+        public static final Distance kElevatorMinHeight = Meters.of(Inches.of(0.0).in(Meters));
+
+        /** The maximum height of the elevator in Meters */
+        public static final Distance kElevatorMaxHeight = Meters.of(Inches.of(74.5).in(Meters));
+
+        /** The height of the elevator in Meters when the robot is in its starting configuration */
+        public static final Distance kElevatorStartingHeight = Meters.of(Inches.of(0.0).in(Meters));
+
+        /** The gearing of the wrist gearbox */
+        public static final double kWristGearing = 64.0;
+
+        /** The weight in Kg of the wrist in Kilograms */
+        public static final Mass kWristMass = Kilograms.of(0.5);
+
+        /** Arm length in meters */
+        public static final Distance kArmLength = Meters.of(Inches.of(12.0).in(Meters));
+
+        /** Min angle in Degrees */
+        public static final Angle kMinAngle = Degrees.of(0);
+
+        /** Max angle in Degrees */
+        public static final Angle kMaxAngle = Degrees.of(180);
+    }
+
+    /** Constants used for interfacing with the Coral Intake and Wrist subsystems */
+    public static final class CoralMechanismConstants {
+        /** The motor ID of the Spark Max associated with the wrist motor for the coral mechanism */
+        public static final int kWristMotorId = 43;
+
+        /** The motor ID of the Spark Max associated with the motor for the coral intake */
+        public static final int kWheelMotorId = 44;
+
+        /* Wrist Configurations */
+        /** The angle that the wrist will be at the start of the match */
+        public static final Angle kStartingAngle = Degrees.of(0);
+
+        /** The angle that the wrist will be when intaking a coral */
+        public static final Angle kIntakingAngle = Degrees.of(37.5);
+
+        /** The angle that the wrist will be when scoring at L4 */
+        public static final Angle kScoringL4 = Degrees.of(120);
+
+        /** The angle that the wrist will be when scoring at L3 */
+        public static final Angle kScoringL3 = Degrees.of(110);
+
+        /** The angle that the wrist will be when scoring at L2 */
+        public static final Angle kScoringL2 = Degrees.of(110);
+
+        /** The angle that the wrist will be when scoring at L1 */
+        public static final Angle kScoringL1 = Degrees.of(120);
+
+        /* PIDF constants */
+        public static final double kP = 1.5;
+        public static final double kI = 0.0;
+        public static final double kD = 0.05;
+        public static final double kF = 0.0;
+
+        /* Wheel Speeds */
+        /** Speed of the motor when intaking a coral piece */
+        public static final double kWheelSpeedIntaking = -0.45;
```

Comment:
Change all of these to Velocities and use Velocity closed loop control instead of open loop control